experimental/examples/falling-balls-mpp/build.gradle.kts

```diff
@@ -163,6 +163,11 @@ compose.experimental {
                 //Usage: ./gradlew iosDeployIPadDebug
                 device = IOSDevices.IPAD_MINI_6th_Gen
             }
+            connectedDevice("Device") {
+                //Usage: ./gradlew iosDeployDeviceRelease
+                // You can specify your teamId here, or in local.properties (compose.ios.teamId=***)
+                teamId = "JMS9FA69HB"
```

Comment:
Is it random digits and symbols? How does it affect deploying?

Reply:
For now it's my teamId value for sample. But users need to change it to their teamId.

Reply:
Is it a secret or public information? What happens if we omit this id or write fake one?
Is there are any validation? For example, if this id will be changed/removed in your account, we will have an error?

If is there any online validation, better to not add a real id here. It will be hard to support, and I am afraid of consequences, because any user can deploy app with your team id.

If we need it for an easy development, we can read this id from Gradle properties, and write it in the global gradle.properties

Reply:
Done

Reply:
And in the other place :)